content/rc/subscriptions/view-flexible-subscription.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: View Flexible subscription details
+description:
+weight: 40
+alwaysopen: false
+categories: ["RC"]
+linktitle: View Flexible subscription
+aliases: 
+---
+To view the details of a Flexible subscription:
+
+1.  Use the admin console menu to select the **Subscriptions** command.  This displays the **Subscriptions** list.
+
+1.  Select the subscription you wish to view.
+
+    {{<image filename="images/rc/subscription-view-flexible.png" width="75%" alt="The View Subscription screen for Flexible subscriptions." >}}{{< /image >}}
+
+Two tabs are available:
+
+1.  The **General** tab summarizes the your subscription details, the cost estimate, and the payment method.
+    
+    **General details** include:
+    
+    | _Detail_ | _Description_ |
+    |:---------|:--------------|
+    | **Subscription name** | A descriptive name for the plan.  Can be changed at any time. |
+    | **Runs on** | Either _RAM_ or _RAM+Flash_ |
+    | **Cloud account** | (_AWS only_) Descriptive name of the associated cloud account.  Useful for organizations that manage [multiple accounts]({{<relref "/rc/how-to/view-edit-cloud-account.md">}}). |
+    | **Cloud** | Describes the subscription cloud provider. |
+    | **Region** | Deployment region details. | 
+    | **Multi-AZ** | Indicates whether replication is deployed across multiple availability zones within the region. |
```

Comment:
This table put periods at the end of sentence fragments. I think that goes against our style guides.

Reply:
Removed.